pkg/disk/nodeserver.go

```diff
@@ -118,15 +120,15 @@ type QueryResponse struct {
 
 // NewNodeServer creates node server
 func NewNodeServer(d *csicommon.CSIDriver, c *ecs.Client) csi.NodeServer {
-	var maxVolumesNum int64 = MaxVolumesPerNode
+	var maxVolumesNum int64 = DefaultMaxVolumesPerNode
 	volumeNum := os.Getenv("MAX_VOLUMES_PERNODE")
 	if "" != volumeNum {
 		num, err := strconv.ParseInt(volumeNum, 10, 64)
 		if err != nil {
 			log.Log.Fatalf("NewNodeServer: MAX_VOLUMES_PERNODE must be int64, but get: %s", volumeNum)
 		} else {
-			if num < 0 || num > 64 {
-				log.Log.Errorf("NewNodeServer: MAX_VOLUMES_PERNODE must between 0-15, but get: %s", volumeNum)
+			if num < 0 || num > int64(MaxVolumesPerNodeLimit) {
```

Comment:
```suggestion
			if num < 0 || num > MaxVolumesPerNodeLimit {
```
consts are untyped, I think we don't need the conversion